ocp_resources/network_attachment_definition.py

```diff
@@ -197,3 +197,42 @@ def to_dict(self):
     @property
     def resource_name(self):
         return f"ovs-cni.network.kubevirt.io/{self.bridge_name}"
+
+
+class FlatL2NetworkAttachmentDefinition(NetworkAttachmentDefinition):
+    def __init__(
+        self,
+        network_name=None,
+        teardown=True,
+        *args,
+        **kwargs,
+    ):
+        """
+        Create and manage a flat-l2 NetworkAttachmentDefinition.
+        API reference:
```

Comment:
- please update docstring, it is not `flat-l2` 
- please use proper docstring formatting

Reply:
Ack

Reply:
Also update the PR title